Doc/library/constants.rst

```diff
@@ -57,6 +57,9 @@ A small number of constants live in the built-in namespace.  They are:
       it currently evaluates as true, it will emit a :exc:`DeprecationWarning`.
       It will raise a :exc:`TypeError` in a future version of Python.
```

Comment:
This can be removed. And maybe the whole text of the note can be reworded.

Reply:
Thanks, I'll submit another PR addressing your comments.